Backend/lib/message-service/src/Lib/API/UI/Message.hs

```diff
@@ -0,0 +1,46 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Lib.API.UI.Message where
```

Comment:
Why your lib is restricted for UI? it operates with metainfo about our main app structure, which doesn't make sense

Reply:
Correct. @SanskarGubreley Resolve this too.